actstream/templatetags/activity_tags.py

```diff
@@ -94,7 +94,7 @@ def render_result(self, context):
             'actstream/%s/action.html' % action_instance.verb.replace(' ', '_'),
             'actstream/action.html',
         ]
-        return render_to_string(templates, {'action': action_instance})
+        return render_to_string(templates, context.flatten())
```

Comment:
could you explain what does context.flatten do?

Reply:
It turns the full context object into a python `dict`. `'action'` is verified to be in here, I've inspected it in the debugger thoroughly.
https://docs.djangoproject.com/en/1.11/ref/templates/api/#django.template.Context.flatten

Reply:
is it possible to verify the changes with some test?

Reply:
Sorry about the late response. I'm not sure how your testing is set up, I tried but was unsuccessful in running the tests you already have. Am I supposed to set up a dummy server outside of the repo and install `actstream` like another app?